samcli/lib/bootstrap/bootstrap.py

```diff
@@ -73,6 +82,23 @@ def _get_stack_template():
                       - "/*"
                 Principal:
                   Service: serverlessrepo.amazonaws.com
+              -
+                Action:
+                  - "s3:*"
+                Effect: "Deny"
+                Resource:  
```

Comment:
According to [here](https://aws.amazon.com/premiumsupport/knowledge-center/s3-bucket-policy-for-config-rule/), do we also need to add the bucket arn in addition to all objects under the bucket?
For example:
```
      "Resource": [
        "arn:aws:s3:::DOC-EXAMPLE-BUCKET",
        "arn:aws:s3:::DOC-EXAMPLE-BUCKET/*"
      ]
```

Reply:
I agree, my bad. Fixed in https://github.com/aws/aws-sam-cli/pull/2953/commits/b241585acfe391f06c581290464da1888063255c

Reply:
Thanks!